Dockerfile-linux.template

```diff
@@ -170,7 +170,10 @@ RUN set -eux; \
 # https://github.com/alpinelinux/aports/commit/2026e1259422d4e0cf92391ca2d3844356c649d0
 		EXTRA_CFLAGS="-DTHREAD_STACK_SIZE=0x100000" \
 {{ ) else "" end -}}
+{{ if is_slim or is_alpine then ( -}}
+# strip all symbols to reduce binaries size
```

Comment:
Let's remove this comment -- I think the flag name is pretty clear already, so this isn't necessary:

```suggestion
```

Reply:
Sure! Updated the template and re-ran `apply-template.sh`.